File: config/config-sbc2ha/src/main/java/com/dfi/sbc2ha/config/sbc2ha/definition/sensor/Lm75SensorConfig.java
package com.dfi.sbc2ha.config.sbc2ha.definition.sensor;

import com.dfi.sbc2ha.config.sbc2ha.definition.enums.PlatformType;

public class Lm75SensorConfig extends ThermBusSensorConfig {

    public Lm75SensorConfig() {
        super();
        platform = PlatformType.LM75;
    }
}

File: config/editor/src/app/definition/sensor/Lm75SensorConfig.ts
import {ThermBusSensorConfig} from "./ThermBusSensorConfig";
import {PlatformType} from "../enums/PlatformType";

export class Lm75SensorConfig extends ThermBusSensorConfig {

  constructor(name: string, busId: string) {
    super(PlatformType.LM75, name, busId);
  }
}
